src/Esfa.Vacancy.Register.Api/Esfa.Vacancy.Register.Api.csproj

```diff
@@ -25,6 +25,7 @@
     <NuGetPackageImportStamp>
     </NuGetPackageImportStamp>
     <TargetFrameworkProfile />
+    <Use64BitIISExpress />
```

Comment:
Visual Studio 2017 bits should be removed as our build servers are using 2015

Reply:
VSTS uses VS 2017